### PR TITLE
added Dave's Mac build installation steps with homebrew to Mac installation page

### PR DESCRIPTION
**Pull request by bartvanosnabrugge**
With the latest Mac OS (since Catalina) there are issues with MacPorts. Hence, this installation guide written by Dave Casson to use home-brew is a welcome addition to the Mac Installation page.

Make sure all the relevant boxes are checked (and only check the box if you actually completed the step):

- [ ] ReleaseNotes entry

### CODE REVIEW

File: docs/installation/SUMMA_on_OS_X.md [hunk 1]
```diff
@@ -62,4 +65,63 @@ In the following I will assume that you don't have a Fortran compiler or NetCDF
...
+
+ ## Instructions with homebrew
+
+ These instructions deviate from the standard SUMMA on OS X installation instructions in that Homebrew is used as an alternative
+ to MacPorts as a package installer. This is largely due to Catalina changes to a users /opt/ folder.
+
+ These instructions assume that you've already cloned the SUMMA repository.
+ If you haven't, follow the initial [SUMMA Installation Instructions](https://summa.readthedocs.io/en/latest/installation/SUMMA_installation/)
+
+ ### Step 1: Installation of Homebrew
```
Comment (wknoben) [inline docs/installation/SUMMA_on_OS_X.md:78]: Can you ensure consistent formatting of steps for both the MacPorts (currently numbered list) and Homebrew (currently sub-headings) sections?

File: docs/installation/SUMMA_on_OS_X.md [hunk 2]
```diff
@@ -62,4 +65,63 @@ In the following I will assume that you don't have a Fortran compiler or NetCDF
...
+
+ If you are already using MacPorts, it is recommended to uninstall it to avoid package issues.
+ Uninstallation instructions are available [here](https://guide.macports.org/chunked/installing.macports.uninstalling.html).
+ If you'd like to save the currently installed packages for MacPorts to file, use the command: `port installed > ports_installed.txt`
+
+ Homebrew can then be installed following the instructions [here](https://brew.sh/)
+
+ ### Step 2: Installation of Required packages
```
Comment (wknoben) [inline docs/installation/SUMMA_on_OS_X.md:86]: As above

File: docs/installation/SUMMA_on_OS_X.md [hunk 3]
```diff
@@ -62,4 +65,63 @@ In the following I will assume that you don't have a Fortran compiler or NetCDF
...
+ Note that as a bonus, NetCDF Fortran is installed with this command and does not require action.
+ It is installed as a [resource](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/netcdf.rb), by default.
+
+ #### LAPACK and openblas
+
+ Use the following commands to install lapack and openblas using Homebrew
+ `brew install lapack`
+ `brew install openblas`
+
+  ### Step 3: Create the Makefile for use in Compliation
```
Comment (wknoben) [inline docs/installation/SUMMA_on_OS_X.md:107]: Comment 1: pleas ensure consistency of step numbering between Macports and Homebrew sections.

Comment (wknoben) [inline docs/installation/SUMMA_on_OS_X.md:107]: Comment 2: Macports section ends with a reference to the general SUMMA install instructions. Suggest to remove this section on how to use a Makefile here and refer to the general page instead, unless these instructions are specific to Homebrew.

**Review by wknoben — CHANGES_REQUESTED** [review]
Hi Bart, thanks for the PR. One general question: did you compile the docs locally and do they come out looking OK?

File: docs/installation/SUMMA_on_OS_X.md (hunk 1 above)
Comment (bartvanosnabrugge) [reply docs/installation/SUMMA_on_OS_X.md:82]: Added sub-headings to MacPorts part, and numbered list to Homebrew part

File: docs/installation/SUMMA_on_OS_X.md (hunk 3 above)
Comment (bartvanosnabrugge) [reply docs/installation/SUMMA_on_OS_X.md]: Yes, these are the environment variables that work with the Homebrew install

Comment (wknoben) [reply docs/installation/SUMMA_on_OS_X.md]: Are these specific to Homebrew though? If not, then it is cleaner to just refer to the main installation page (like is done in the Macports section).

**Comment by wknoben** [comment]
Two more questions (copied from above):

1. Did you compile the docs locally and do they come out looking OK?
2. Could you also update the Release Notes entry?

**Review by wknoben — CHANGES_REQUESTED** [review]
See comment re. compiling/environment variables and general comment

File: docs/installation/SUMMA_on_OS_X.md (hunk 3 above)
Comment (bartvanosnabrugge) [reply docs/installation/SUMMA_on_OS_X.md]: Yes, these work specifically with Homebrew. The instal requirements on the installation page are written in a generalized form, these give the precise values for this install with home-brew.

**Comment by bartvanosnabrugge** [comment]
Closed due to new pull request to develop branch which supersedes this one

**Pull request closed without merging**